nengo_viz/static/viz_slider.js

```diff
@@ -151,6 +127,34 @@ VIZ.Slider = function(args) {
 VIZ.Slider.prototype = Object.create(VIZ.Component.prototype);
 VIZ.Slider.prototype.constructor = VIZ.Slider;
 
+VIZ.Slider.prototype.set_value = function(graph, slider, value) {
+    /** check if click was the middle mouse button */
+    var target = slider;
+    var self = graph;
+    var slider_index = target.slider.index; // Get index (For 1D > sliders)
```

Comment:
Hmm, this feels weird to me.  I'd pass in the index and get the slider with ```this.sliders[index]```.  It just seems odd to me to pass in the object and figure out its index, rather than passing in the index and figuring out the object.